packages/drivers/odsp-urlResolver/src/urlResolver.ts

```diff
@@ -36,6 +39,10 @@ export class OdspUrlResolver implements IUrlResolver {
 	}
 }
 
+// eslint-disable-next-line jsdoc/require-description
```

Comment:
Does it complain without this line? Seems like the class above didn't and it also doesn't have description.

Reply:
![image](https://github.com/microsoft/FluidFramework/assets/111468570/ee74abd0-ede2-427a-852e-a542ccf522e9)

Linter is weird sometimes 😞. Doesn't complain about the missing description for the class but does so for the function. The function was pretty straightforward, so I added the description instead.